### DISCUSSION TITLE
Unable to update Adguard via web gui

### DISCUSSION THREAD

**Discussion by notorioustubze** [tteck/Proxmox#711] [Help Needed] [answered]
Hi Tteck, my Adguard instance has been running fine for some time however recently i'm unable to upgrade via the web gui. I get the error message "Auto-update-failed please upgrade manually" I've tried serval time with the guide they provide but I can't figure it out. Are you able to assist?

**Comment by tteck** [comment, marked as answer]
See if this works. Run in the Adguard LXC console.
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/adguard-update.sh)"
```

**Reply by notorioustubze** [reply]
Worked perfectly! Thank you so much :)
